changelog/unreleased/3296

```diff
@@ -0,0 +1,6 @@
+Enhancement: Hide more section if all options are disabled
```

Comment:
I'd do a small change here: `Hide "More" section in Settings ... `.  Quotation marks help to distinguish the literal.